src/ResourceManager/PowerBIDedicated/AzureRM.PowerBIDedicated.psd1

```diff
@@ -0,0 +1,132 @@
+#
+# Module manifest for module 'PSGet_AzureRM.PowerBIDedicated'
```

Comment:
@selasagi sent another email about changing the name of this module to `AzureRM.PowerBI`

Reply:
done